packages/typespec-ts/test/integration/generated/parameters/collection-format/src/index.d.ts

```diff
@@ -87,14 +117,20 @@ export declare interface QueryPipes204Response extends HttpResponse {
     status: "204";
 }
 
+export declare interface QueryPipesColorsQueryParam {
+    value: string[];
+    explode: false;
+    style: "pipeDelimited";
+}
+
 export declare type QueryPipesParameters = QueryPipesQueryParam & RequestParameters;
 
 export declare interface QueryPipesQueryParam {
     queryParameters: QueryPipesQueryParamProperties;
 }
 
 export declare interface QueryPipesQueryParamProperties {
-    colors: string;
+    colors: QueryPipesColorsQueryParam;
```

Comment:
why this is not show as compatibleMode?

Reply:
changed the option to compatibilityQueryMultiFormat.